### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by Ja-Xian-Chen**
Changed font to Koulen
Cleared unnecessary files

### CODE REVIEW

File: .gitignore
Comment (adruzenko03) [inline .gitignore:1]: gitignore is necessary, please add back

File: package.json
Comment (adruzenko03) [inline package.json:1]: Package.json is necessary, please add back

File: package-lock.json
Comment (adruzenko03) [inline package-lock.json:1]: Package Lock also important, please add back

**Review by adruzenko03 — CHANGES_REQUESTED** [review]
Please undo the "delete unnecessary files" commit, as a lot of those files are very necessary

**Comment by Ja-Xian-Chen** [comment]
Added files back

**Pull request closed without merging**